fw/t/unit/test_http_parser.c

```diff
@@ -203,6 +280,38 @@ do_split_and_parse(unsigned char *str, int type)
 	return r;
 }
 
+static int
+do_h2_parse(unsigned char *str, unsigned int len, int type)
+{
+	int r;
+	unsigned int parsed;
+
+	BUG_ON(!str);
+
+	if (type == FUZZ_REQ_H2) {
+		if (req)
+			test_req_free(req);
+
+		req = test_req_alloc(len);
+	}
+	else {
+		BUG();
+	}
+
+	req->conn = (TfwConn*)&conn;
+	req->pit.parsed_hdr = &h2_parsed_hdr;
+	stream.msg = (TfwMsg*)req;
+	req->stream = &stream;
+
+	r = tfw_h2_parse_req(req, str, len, &parsed);
+
+	((TfwHttpMsg*)req)->msg.len += parsed;
+	if (r == T_POSTPONE)
+		r = tfw_h2_parse_req_finish(req);
+
+	return r;
+}
+
```

Comment:
Basically all introduced code h2 specific code in http parser unit tests is only a workaround around broken chunked parsing. It can be discarded immidiatly after chunked h2 frame parsing fixed.

May be there is not need for separate implementation for h2 parsing for now?